app/packages/map/src/Map.tsx

```diff
@@ -190,7 +193,10 @@ const Plot: React.FC<{}> = () => {
               if (draw.getMode() !== "draw_polygon") {
                 draw.changeMode("draw_polygon");
               }
-            } catch {}
+            } catch (error) {
+              console.error(error);
+              setMapError(true);
```

Comment:
Maybe the below will be easier for people to debug?
```suggestion
              setMapError(true);
              throw error
```

Reply:
👍